deploy/helm_charts/mixer/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default values for mixer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Name of the Chart.
+nameOverride: ""
+# Full name of the release in the following format.
+# <release name>-<chart name>
+fullnameOverride: ""
+
+# Namespace for the deployment.
+# Set enabled to false to deploy into an existing namespace.
+namespace:
+  create: false
+  name: "mixer"
+
+# Mixer service config.
+mixer:
+  gcpProjectID:
+  githash:
+
+  useTMCFCSVData: false
+  # Name of the GCS bucket without scheme.
+  # Example: datcom-public
+  tmcfCSVBucket:
+  # Folder under the CSV bucket.
+  tmcfCSVFolder:
+
+  
+  # If bigqueryOnly is True, bigqueryTableRef must be specified.
+  bigqueryOnly: false
+  # Example: google.com:datcom-store-dev.dc_v3_encode_clustered
+  bigqueryTableRef: 
+
+  # URL root for the mixer service launched.
+  # Example: dev.api.datacommons.org
+  serviceName: 
+ 
+  image:
+    repository: gcr.io/datcom-ci/datacommons-mixer
+    pullPolicy: IfNotPresent
+    tag: "latest"
+
+  schemaConfigs:
+    base.mcf: 
+    dailyweather.mcf:
+    monthlyweather.mcf:
+    encode.mcf: 
+
+esp:
+  image:
+    repository: gcr.io/endpoints-release/endpoints-runtime
+    pullPolicy: IfNotPresent
+    tag: "1"
+
+# Config for k8s-sa(service account). The k8s-sa will be bound to a
+# GCP-sa using annotations, specified below.
+serviceAccount:
+  create: false
+  # K8s service account name.
+  name: mixer-ksa
+  # Google service account name.
+  gsaName: mixer-robot
+
+ingress:
+  name: mixer-ingress
+  enabled: true
+  annotations: {
+    kubernetes.io/ingress.global-static-ip-name: mixer-ip,
+    ingress.gcp.kubernetes.io/pre-shared-cert: mixer-certificate
+  }
+
+kgStoreConfig:
+  bigqueryVersion:
+  bigtableImportGroupsVersion:
+  storeProjectID: datcom-store
+
+memdbJSON: ""
+
+serviceGroups:
+  - svg: 
+      urlPaths:
+        - "/place/stat-var-group/*"
+        - "/stat-var/*"
+        - "/v1/info/variable/group/*"
```

Comment:
Please check with Bo if this recently changed (from most recent PR).

Reply:
Will check this in for now for testing and split again in a follow up CL.
This change will not affect any CI systems. Follow up CLs will change deploy_gke.sh, which is when the helm changes go live.